Month1/Week2/Section1/Exercise4/Main4.cpp

```diff
@@ -23,31 +23,28 @@ o
 */
 #include<iostream>
 #include<string>
-class Printer
+class FPrinter
 {
 private:
 	std::string OutputString;
 public:
-	Printer()
+	FPrinter()
 	{
 	}
-	Printer(std::string OutputString)
-	{
-		OutputString = "";
-	}
+	
```

Comment:
Why are you removing the other constructors? They are required by the exercise specifications.

Reply:
Added